anchor/anchorparser.go

```diff
@@ -37,13 +37,14 @@ func ParseAnchors(protoAnchors []*yotiprotoattr.Anchor) []*Anchor {
 					value string
 					err   error
 				)
-				anchorType, value, err = parseExtension(ext)
+				parsedAnchorType, value, err := parseExtension(ext)
```

Comment:
I can't remember 100%, but I think we need to consider the case of parsing actual Unknown anchors.  I think this currently would just abandon an unknown anchor (instead of just the critical extension).  Do we have any tests for unknown anchors in Go?

Reply:
TestAnchorParser_UnknownAnchor (yoti_test.go:973)